### ISSUE TITLE
crab getlog broken in latest version

### ISSUE THREAD

**Issue by belforte** [dmwm/CRABServer#6598]
```
ERROR 2021-05-17 21:08:18.693 UTC: 	 Fatal error trying to connect to https://cmsweb-testbed.cern.ch:8443/crabserver/preprod/task using {'subresource': 'webdirprx', 'workflow': u'210517_155933:belforte_crab_20210517_175924'}
DEBUG 2021-05-17 21:08:18.695 UTC: 	 Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-dev/v3.210505.00/lib/ServerUtilities.py", line 201, in getProxiedWebDir
    dictresult, _, _ = crabserver.get(api='task', data=data)  # the second and third parameters are deprecated
  File "/cvmfs/cms.cern.ch/share/cms/crab-dev/v3.210505.00/lib/RESTInteractions.py", line 212, in get
    return self.server.get(uri, data)
  File "/cvmfs/cms.cern.ch/share/cms/crab-dev/v3.210505.00/lib/RESTInteractions.py", line 99, in get
    return self.makeRequest(uri=uri, data=data, verb='GET')
  File "/cvmfs/cms.cern.ch/share/cms/crab-dev/v3.210505.00/lib/RESTInteractions.py", line 149, in makeRequest
    verbose=self['verbose'])
  File "/cvmfs/cms.cern.ch/share/cms/crab-dev/v3.210505.00/lib/WMCore/Services/pycurl_manager.py", line 275, in request
    raise exc
HTTPException: url=https://cmsweb-testbed.cern.ch:8443/crabserver/preprod/task, code=400, reason=Bad Request, headers={'Content-Length': '733', 'X-Error-Http': '400', 'X-Rest-Time': '1534.939 us', 'Server': 'CherryPy/17.4.0', 'Connection': 'close', 'X-Error-Detail': 'Invalid input parameter', 'CMS-Server-Time': 'D=24285 t=1621285698667145', 'X-Rest-Status': '302', 'Date': 'Mon, 17 May 2021 21:08:18 GMT', 'Content-Type': 'text/html;charset=utf-8', 'X-Error-Info': "Incorrect 'subresource' parameter", 'X-Error-Id': 'ae02cd72db9ff92ca314d287ca3891bd'}

```

code about `webdirprx` has not changed . And the validation list contains just `webdir` like before
https://github.com/dmwm/CRABServer/blob/d19050fc571e1c1577e3436013d716535106b040/src/python/CRABInterface/Regexps.py#L82

maybe WMCore validation has become more strict and while before `webdirprx` was matching `webdir`, now it does not anymore ?
I can agree that it is better to list explicitly the possible subresource values, but need to make sure I understand before changing code.

**Comment by belforte** [comment]
FWIW, I hacked cmsweb-testbed adding `webdirprx` to the list in 
https://github.com/dmwm/CRABServer/blob/d19050fc571e1c1577e3436013d716535106b040/src/python/CRABInterface/Regexps.py#L82
and `crab getlog --short --jobids=1` works again
```
belforte@lxplus781/TC3> crab getlog --short --jobids=1
Setting the destination to /afs/cern.ch/work/b/belforte/CRAB3/TC3/crab_20210517_175924/results 
Retrieving...
Retrieved job_out.1.0.txt
Success: All files successfully retrieved.
Log file is /afs/cern.ch/work/b/belforte/CRAB3/TC3/crab_20210517_175924/crab.log
belforte@lxplus781/TC3> 

```

**Comment by belforte** [comment]
OK. There was a double bug.
Original code
https://github.com/dmwm/CRABServer/blob/d19050fc571e1c1577e3436013d716535106b040/src/python/CRABInterface/Regexps.py#L82
* bug 1 : `webdirprx` was missing
* bug 2: the list of possible subresource strings in `OR` is not inside `(...)` so the final `$` only applies to the `updatepublicationtime` string which is not allowed to have anything else at its right, all other strings match as long as the `subresource` param in the HTTP query starts with one of them

Bug2 made people not notice bug1.
When I fixed bug2 in https://github.com/dmwm/CRABServer/pull/6580, I triggered bug 1.
